Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs

```diff
@@ -0,0 +1,450 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System;
+
+/// <summary>
+/// A conglimeration of open-source simplex libraries in C# with an emphasis on performance
+/// </summary>
+public class FastSimplexNoise
+{
+    private const double STRETCH_2D = -0.211324865405187;
+    private const double STRETCH_3D = -1.0 / 6.0;
```

Comment:
Why do the math if we know what the value is going to be?

Reply:
This tells you that the value should probably be normalized:
const float SOME_NORMALIZED_VALUE = 1f / 25;

This is a little more vague:
const float SOME_MYSTERIOUS_VALUE = 0.04f;

I'll update the other numbers to follow the same 1 / x format, for consistency.